### DISCUSSION TITLE
when i click on build server on postal it not build the mail server<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by umersab** [community/community#130739] [Programming Help]
### Select Topic Area

Bug

### Body

when i click on build server on postal it not build the mail server

**Comment by LiteBrite82** [comment]
Hi @umersab ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Postalserver repository](https://github.com/postalserver/postal/issues/1251). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
